applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.hpp

```diff
@@ -365,12 +236,12 @@ class UpdatedLagrangian
     /**
      * Sets on rElementalDofList the degrees of freedom of the considered element geometry
      */
-    void GetDofList(DofsVectorType& rElementalDofList, ProcessInfo& rCurrentProcessInfo) override;
+    void GetDofList(DofsVectorType& rElementalDofList, const ProcessInfo& rCurrentProcessInfo) const override;
```

Comment:
nice, thx for doing this without being asked :D

Reply:
❤️  phil loves const ❤️